submodules/k8s/main.tf

```diff
@@ -23,7 +23,7 @@ locals {
         aws_auth_yaml       = basename(local.aws_auth_filename)
         eniconfig_yaml      = local.eniconfig_filename != "" ? basename(local.eniconfig_filename) : ""
         calico_operator_url = local.calico.operator_url
-        bastion_user        = var.bastion_user != null ? var.bastion_user : ""
+        bastion_user        = coalesce(var.bastion_user, "")
```

Comment:
what is the answer when var.bastion_user is null or ""? coalesce ignores both null and "".
if this is the answer then why aren't we doing the same for bastion_public_ip?